.github/workflows/tests.yml

```diff
@@ -169,3 +169,209 @@ jobs:
+        with:
+          coverage: "none"
+          ini-values: "memory_limit=-1"
+          php-version: "7.4"
+          extensions: amqp, apcu, igbinary, intl, mbstring, memcached, mongodb, redis
+          tools: flex
+
+      - name: Configure composer
+        run: |
+          SYMFONY_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*')
+          SYMFONY_REQUIRE=">=$SYMFONY_VERSION"
+          COMPOSER_HOME="$(composer config home)"
+          ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
+
+          echo COMPOSER_ROOT_VERSION=$SYMFONY_VERSION.x-dev >> $GITHUB_ENV
+          echo SYMFONY_VERSION=$SYMFONY_VERSION >> $GITHUB_ENV
+          echo SYMFONY_REQUIRE="$SYMFONY_REQUIRE" >> $GITHUB_ENV
+
+      - name: Install PHPUnit
+        run: |
+          mkdir -p ./vendor/symfony
+          cd ./vendor/symfony
+          ln -s ../../src/Symfony/Bridge/PhpUnit phpunit-bridge
+          cd ../..
+          ./phpunit install
+          echo PHPUNIT=`pwd`/phpunit >> $GITHUB_ENV
+
+      - name: Run tests
+        run: |
+          _run_tests() {
+            ok=0
+            echo "::group::$1"
+
+            # Install dependencies
+            cd ./$1
+            composer update --no-progress --ansi 2>&1 || ok=1
+
+            if [ $ok -eq 0 ]; then
+              # Run the tests
+              $PHPUNIT --colors=always --exclude-group tty,benchmark,intl-data,legacy 2>&1 || ok=1
+            fi
+
+            echo ""
+            echo "::endgroup::"
+
+            if [ $ok -ne 0 ]; then
+              echo "::error::$1 failed"
+            fi
+
+            return $ok
+          }
+          export -f _run_tests
+
+          find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -not -wholename '*/Bridge/PhpUnit/*' -print0 | xargs -0 -n1 dirname | sort | parallel _run_tests
+
+  low_deps:
+    name: PHPUnit (8.0, low deps)
+    runs-on: Ubuntu-20.04
+
+    env:
+      SYMFONY_DEPRECATIONS_HELPER: weak
+      SYMFONY_REQUIRE: ">=2.3"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          ini-values: "memory_limit=-1"
+          php-version: "8.0"
+          # TODO: memcached is excluded because of compatibility issues with PHP 8
+          extensions: amqp, apcu, igbinary, intl, mbstring, :memcached, mongodb, redis
+          tools: flex
+
+      - name: Configure composer
+        run: |
+          SYMFONY_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*')
+          COMPOSER_HOME="$(composer config home)"
+          ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
+
+          echo COMPOSER_ROOT_VERSION=$SYMFONY_VERSION.x-dev >> $GITHUB_ENV
+          echo SYMFONY_VERSION=$SYMFONY_VERSION >> $GITHUB_ENV
```

Comment:
Is it an idea to use Yaml anchors to avoid all the code duplication?

E.g.

```yaml
steps:
  - name: Checkout
    uses: actions/checkout@v2

  - name: Setup PHP
    uses: shivammathur/setup-php@v2
    with:
      <<: *php_variables
      # TODO: memcached is excluded because of compatibility issues with PHP 8
      extensions: amqp, apcu, igbinary, intl, mbstring, :memcached, mongodb, redis

  - *configure_composer
```

Reply:
Maybe. I've removed the high/low deps jobs for now. That should eliminate a lot of C&P as well.

Reply:
@wouterj Do YAML anchors work in GHA? Last time I checked they did not...